crates/biome_js_analyze/tests/specs/nursery/noHeadImportInDocument/pages/_document/index.jsx.snap

```diff
@@ -0,0 +1,23 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+assertion_line: 86
+expression: index.jsx
+---
+# Input
+```jsx
+import Head from "next/head";
+```
+
+# Diagnostics
+```
+index.jsx:1:1 lint/nursery/noHeadImportInDocument ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Don't use next/head in pages/document.js
```

Comment:
The is incorrect because the analyzed file is `document/index.jsx`. Wouldn't it be more accurate to use the real file name instead?